indy_node/server/request_handlers/domain_req_handlers/claim_def_handler.py

```diff
@@ -0,0 +1,68 @@
+from indy_common.auth import Authoriser
+
+from indy_common.state import domain
+
+from indy_common.constants import CLAIM_DEF, REF, SCHEMA
+
+from plenum.common.constants import DOMAIN_LEDGER_ID
+from plenum.common.exceptions import UnauthorizedClientRequest, UnknownIdentifier, InvalidClientRequest
+from plenum.common.request import Request
+from plenum.common.roles import Roles
+from plenum.common.txn_util import get_type
+from plenum.server.database_manager import DatabaseManager
+from plenum.server.request_handlers.handler_interfaces.write_request_handler import WriteRequestHandler
+
+
+class ClaimDefHandler(WriteRequestHandler):
+
+    def __init__(self, database_manager: DatabaseManager):
+        super().__init__(database_manager, CLAIM_DEF, DOMAIN_LEDGER_ID)
+
+    def static_validation(self, request: Request):
+        pass
+
+    def dynamic_validation(self, request: Request):
```

Comment:
Please use the latest code from master with the latest Validator approach for every txn type

Reply:
done